packages/mask/src/plugins/Predict/Azuro/components/EventDate.tsx

```diff
@@ -0,0 +1,23 @@
+import { Grid, Typography } from '@mui/material'
+import { makeStyles } from '@masknet/theme'
+
+const useStyles = makeStyles()((theme) => ({
+    date: { fontSize: 14, fontWeight: 300 },
+}))
+
+interface EventDateProps {
+    date: number
+}
+
+export function EventDate(props: EventDateProps) {
+    const { date } = props
+    const { classes } = useStyles()
+
+    const dateToUTC = new Date(date).toUTCString()
```

Comment:
Each time this component gets rerendered. It will create a new date string. Is this an expected behavior?

Reply:
Yes. It is needed in order to convert each event's starting date to UTC.

Reply:
Done.